src/odemis/driver/test/smaract_test.py

```diff
@@ -274,7 +274,7 @@ def test_move_cancel(self):
         self.dev.moveAbs({'x': 0, 'y': 0, 'z': 0, 'rx': 0, 'rz': 0}).result()
         new_pos = {'x':0.001, 'y': 0, 'z': 0.0007, 'rx': 0.001, 'rz': 0.002}
         f = self.dev.moveAbs(new_pos)
-        time.sleep(0.05)
+        time.sleep(0.01)
         f.cancel()
 
         difference = new_pos['x'] - self.dev.position.value['x']
```

Comment:
Let's add a test case for the position update during move. It should be just a matter of regularly reading the position for eg 1s after asking the move, and checking that you've got at least one position which is different from the initial and target positions.